workshops/collaborative_sketch/README.md

```diff
@@ -278,24 +356,14 @@ function drawPointIfMousePressed() {
 And then use it:
 
 ```js
+̶c̶a̶n̶v̶a̶s̶.̶m̶o̶u̶s̶e̶M̶o̶v̶e̶d̶(̶d̶r̶a̶w̶P̶o̶i̶n̶t̶)̶;̶
 canvas.mouseMoved(drawPointIfMousePressed);
 ```
-
-**OR**
-
-We can use an anonymous function:
-
-```js
-canvas.mouseMoved(function () {
-  if (mouseIsPressed) {
-    drawPoint();
-  }
-});
-```
-
 Notice that the body of the function is the same! The only difference is that our anonymous function has no name, and is created right inside `.mouseMoved()`.
```

Comment:
Think this can be removed -- it was an explanation for https://github.com/hackclub/hackclub/pull/895/files#diff-be0bd94b73f0ec518553801669aaa9aeL283